docs/getting_started/mod_state/index.md

```diff
@@ -1,64 +1,50 @@
-# State Module
+# State Management Module
 
 ## Introduction
 
-The state module features many general purpose scripts related to state management, but to get started you only need to focus on the `CombatStateMachine`, `GraphNodeStateMachineGlobal`, and the `CombatSituationBuilder`.
-
-The `CombatStateMachine` is a node which can contains and switch between multiple situations. In this context a situation just refers to an internal state machine which represents the set of actions available to a combatant. The combat state machine can buffer inputs which are fed to situations to determine what combat state to transition to.
-
-The `CombatSituationBuilder` is just a helper class which makes the construction of situations, represented by the `GraphNodeStateMachineGlobal`, more user-friendly. 
-
-
-## Adding CombatStateMachine node to the scene tree
-
-If the plugin is enabled then the CombatStateMachine should be available from the 'Create New Node' dialog.
-
-![Image of state machine node in create new node window](images/create_new_sm.png)
-
-![Image of state machine in scene tree](images/scene_tree.png)
+The state management module includes a hierarchical combat state machine for managing the state of combatants in your game, as well as a general-purpose state machine for capturing more general state flows. State management is an essential part of designing complex game  mehchanics as it helps define different states a game entity can be in and what actions can be performed in those states. In regards to fighting games that means defining what buttons transition into what attacks, and what attcks are available during any state.
```

Comment:
`complex game mehchanics, as it helps`*
`can be in, and what`*
`In regards to fighting games, that means `*